### PR TITLE
Change routing and order of questions

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

This changes the order and routing of question in household and individual GB-ENG, GB-WLS schemas as per google sheet. To review you can use [this Quick Launcher](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://gist.githubusercontent.com/petechd/1e79694b3a500ff16171f5f579c40557/raw/c9bd0ba5fcb0cd375e843e27149fa2b9a603357e/census_individual_gb_eng.json). The changes is described on [this Trello card](https://trello.com/c/69ivAs02/3606-a-routing-and-question-order-changes-eng-wls-m).

### Checklist 

Check all requested routing and question changes are implemented.

### CODE REVIEW

**Comment by petechd** [comment]
[[A] Routing and Question Order changes (Eng/Wls) (M)](https://trello.com/c/69ivAs02/3606-a-routing-and-question-order-changes-eng-wls-m)

**Comment by iwootten** [comment]
So the content changes aren't applicable to this card - they are all subject to review and covered here: https://trello.com/c/5NWYjTTQ/3517-a-content-and-validation-changes

File: source/jsonnet/england-wales/individual/blocks/identity-and-health/length_of_stay.jsonnet [hunk 1]
```diff
@@ -51,13 +51,13 @@ function(region_code) {
...
         when: [
-          rules.under3,
+          rules.under1,
         ],
       },
     },
     {
       goto: {
-        block: if region_code == 'GB-WLS' then 'understand-welsh' else 'language',
+        block: 'past-usual-household-address',
```
[the remark this answers is not in the document]
Comment (petechd) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/length_of_stay.jsonnet]: My interpretation of this requirement is: "piped 30 day address" or "address outside UK" are both answers in past-usual-household-address so they route you from past-usual-household-address question to national-identity skipping last-year-address. "student-address" and "other UK address" look both like answers in past-usual-household-address as well but they route you the standard way to next question. "under1" makes sense if it's routing from length-of-stay straight to national-identity skipping past-usual-household-address altogether.

[the remark this answers is not in the document]
Comment (iwootten) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/length_of_stay.jsonnet]: Probably worth getting confirmation from @donnap573 on what it should actually be to certain.

[the remark this answers is not in the document]
Comment (ajmaddaford) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/length_of_stay.jsonnet]: I agree with @iwootten, as the comments on the sheet are associated with the length of stay question.

[the remark this answers is not in the document]
Comment (petechd) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/length_of_stay.jsonnet]: Ok, for everyone who is still interested, spreadsheet has been updated, there is a new column now with all the fixes.

File: source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet [hunk 2]
```diff
@@ -90,25 +90,7 @@ function(region_code, census_month_year_date) {
...
-            },
-          },
-          rules.under3,
-        ],
-      },
-    },
-    {
-      goto: {
-        block: if region_code == 'GB-WLS' then 'understand-welsh' else 'language',
+        block: 'past-usual-household-address',
```
Comment (iwootten) [inline source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: Looking through the other routes from this block - I can't see where under 1 is handled?

Comment (iwootten) [inline source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: Isn't under 1 meant to be handled in this blocks routes too?

Comment (petechd) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: Under 1 goes to length-of-stay and then straight to national-identity.

Comment (iwootten) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: But there's no when condition within arrive_in_country to handle that?

Comment (petechd) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: there is, if you are under 1 then you arrived in country less than one year before census date.

Comment (iwootten) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: Ah right, so the 'greater than' rule applies. Not confusing at all.

Comment (petechd) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet]: it does, it was proved empirically in quick launcher.

**Pull request merged**